packages/foreman/rubygem-kafo/rubygem-kafo.spec

```
@@ -34,6 +34,8 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Obsoletes: rubygem-%{gem_name} <= 4.1.0
```

Could you hide this in non-SCL context? True for all files
```suggestion
%{?scl:Obsoletes: rubygem-%{gem_name} <= 4.1.0}
```